documentation/src/test/java/guides/operators/UniJoinTest.java

```diff
@@ -1,28 +1,38 @@
 package guides.operators;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import io.smallrye.mutiny.Uni;
 import io.smallrye.mutiny.groups.UniJoin;
-import org.junit.jupiter.api.Test;
-
-import java.util.Arrays;
+import java.time.Duration;
 import java.util.List;
 import java.util.function.Supplier;
+import org.junit.jupiter.api.Test;
 
 class UniJoinTest {
```

Comment:
It's not the test class you are looking for, this one is for the documentation.

You should add tests to https://github.com/smallrye/smallrye-mutiny/blob/main/implementation/src/test/java/io/smallrye/mutiny/operators/uni/UniJoinTest.java